File: day1/a.py
with open('input.txt', 'r') as f:
    data = f.read().split("\n")

total = 0
result = 0
top = []

for value in data:
    if value:
        total += int(value)
    else:
        top.append(total)
        total = 0

top.sort(reverse=True)
print(sum(top[:3]))

File: day1/a.ts
const input = Deno.readTextFileSync("input.txt");

let max,
  sum = 0;

for (const line of input.split("\r\n")) {
  if (line) {
    sum += Number(line);
    if (!max || max < sum) max = sum;
  } else {
    sum = 0;
  }
}

console.log("MAX: ", max);
